db/dummydata.sql

```diff
@@ -126,13 +126,13 @@ insert into PROMOTIONS (name, description, promo_code, start_date, end_date, sta
 insert into OFFERS (branch_id, promo_id) values (1, 1), (3, 2), (4, 3), (9, 3);
 
 alter sequence reservations_id_seq restart with 1;
-insert into RESERVATIONS (customer_id, branch_id, pax, reservedSlot, reservedDate, promo_used, confirmed) values (1, 1, 3, '10:00:00', current_date, 'P001', false);
-insert into RESERVATIONS (customer_id, branch_id, pax, reservedSlot, reservedDate, promo_used, confirmed) values (2, 1, 3, '12:00:00', current_date, 'P001', false);
+insert into RESERVATIONS (customer_id, branch_id, pax, reservedSlot, reservedDate, promo_used, confirmed) values (2, 1, 3, '10:00:00', current_date, 'P001', false);
```

Comment:
Any reason for changing the reservations dummy data? I changed the customers dummy data so that by default, they have some reservations.

Reply:
done!